fenix/app/src/main/java/org/mozilla/fenix/gleanplumb/state/MessagingMiddleware.kt

```diff
@@ -38,7 +38,7 @@ class MessagingMiddleware(
         when (action) {
             is Restore -> {
                 coroutineScope.launch {
-                    val messages = messagingStorage.getMessages()
+                    val messages = messagingStorage.getAllMessages()
```

Comment:
@rocketsroger this would also be my understanding - e.g. we are restoring the entire list. 

@jonalmeida looks like you originally implemented this, is this correct?

Reply:
This is expecting non expired, eligible messages.

Reply:
I've updated so it only restores non expired eligible messages.